test/transaction_repository_test.rb

```diff
@@ -1,127 +1,127 @@
-      result: 'fail',
-      created_at: '2012-03-27 14:54:09 UTC',
-      updated_at: '2012-03-27 14:54:09 UTC'
-    },
-    {
-      id: 2,
-      invoice_id: 2,
-      credit_card_number: '4580251236515201',
-      credit_card_expiration: '2012-03-27 14:54:09',
-      result: 'success',
-      created_at: '2012-03-27 14:54:09',
-      updated_at: '2012-03-27 14:54:09 UTC'
-    },
-    {
-      id: 3,
-      invoice_id: 3,
-      credit_card_number: '4354495077693036',
-      credit_card_expiration: '2012-03-27 14:54:09',
-      result: 'success',
-      created_at: '2012-03-27 14:54:09 UTC',
-      updated_at: '2012-03-27 14:54:09 UTC'
-    }
+  attr_reader :transaction_repo, :sales_engine
+
+  def setup
+    transactions = [
+      {
+        id: 1,
+        invoice_id: 1,
+        credit_card_number: "4654405418249632",
+        credit_card_expiration: "2012-03-27 14:54:09",
+        result: "fail",
+        created_at: "2012-03-27 14:54:09 UTC",
+        updated_at: "2012-03-27 14:54:09 UTC"
+      },
+      {
+        id: 2,
+        invoice_id: 2,
+        credit_card_number: "4580251236515201",
+        credit_card_expiration: "2012-03-27 14:54:09",
+        result: "success",
+        created_at: "2012-03-27 14:54:09",
+        updated_at: "2012-03-27 14:54:09 UTC"
+      },
+      {
+        id: 3,
+        invoice_id: 3,
+        credit_card_number: "4354495077693036",
+        credit_card_expiration: "2012-03-27 14:54:09",
+        result: "success",
+        created_at: "2012-03-27 14:54:09 UTC",
+        updated_at: "2012-03-27 14:54:09 UTC"
+      }
     ]
 
     @sales_engine = Minitest::Mock.new
-    @transaction_repository = TransactionRepository.new(transactions, sales_engine)
+    @transaction_repo = TransactionRepository.new(transactions, sales_engine)
   end
 
   def test_all
-
-    assert_equal 3, transaction_repository.all.size
+    assert_equal 3, transaction_repo.all.size
   end
 
   def test_random
-
-    assert_class = Transaction, transaction_repository.random
+    assert_equal Transaction, transaction_repo.random.class
   end
 
   def test_find_by_transaction_id
+    transaction = transaction_repo.find_by_id(1)
 
-    transaction = transaction_repository.find_by_id(1)
     assert_equal 1, transaction.id
   end
 
   def test_find_by_invoice_id
+    transaction = transaction_repo.find_by_invoice_id(2)
 
-    transaction = transaction_repository.find_by_invoice_id(2)
     assert_equal 2, transaction.invoice_id
   end
 
   def test_find_by_credit_card_number
+    transaction = transaction_repo
+      .find_by_credit_card_number("4354495077693036")
 
-    transaction = transaction_repository.find_by_credit_card_number('4354495077693036')
-    assert_equal '4354495077693036', transaction.credit_card_number
+    assert_equal "4354495077693036", transaction.credit_card_number
   end
 
   def test_find_by_credit_card_expiration
+    transaction = transaction_repo
+      .find_by_credit_card_expiration("2012-03-27 14:54:09")
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.